client/my-sites/stats/stats-empty-state/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Card } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+
+import './style.scss';
+
+interface Props {
+	stateType: string;
+}
+
+export default function StatsEmptyState( { stateType }: Props ) {
+	const _stateType = String( stateType ).toLowerCase().trim();
```

Comment:
I noticed the capitalization of the first letter (**Likes / Views / Visitors / Comments**) [mentioned in the task](https://github.com/Automattic/wp-calypso/issues/71021) for the selected type, which could highlight the message.

Reply:
@dognose24 I followed the design prototype which is all lowercase.

Reply:
@jeffgolenski can help me with this?

Should the selected tab be lowercase or have its first letter capitalized?

![image](https://user-images.githubusercontent.com/6406071/211345246-e5c18112-eca8-4906-a3f1-06723d5e73c2.png)
![image](https://user-images.githubusercontent.com/6406071/211345157-970d5900-d500-41a9-8627-1f7e22feb131.png)

Reply:
(@wilmersondasilva, feel free to click "Resolve conversation"!)